Source/Components/ImageGlass.Settings/Configs.cs

```diff
@@ -64,6 +64,11 @@ public static class Configs {
         /// </summary>
         public static bool IsRandomSlideshowInterval { get; set; } = false;
 
+        /// <summary>
+        /// Gets, sets value of alert in Slideshow state
+        /// </summary>
+        public static bool IsImgChangeAlert { get; set; } = false;
```

Comment:
`Img` is a class in ImageGlass.Heart, just use full name to avoid confusing

```suggestion
        public static bool IsImageChangeAlert { get; set; } = false;
```

Reply:
done